pulsar-common/src/main/proto/PulsarMarkers.proto

```diff
@@ -85,5 +85,5 @@ message MessageIdData {
 
 /// --- Transaction marker ---
 message TxnCommitMarker {
-    required MessageIdData message_id = 1;
+    repeated MessageIdData message_id = 1;
```

Comment:
```suggestion
    repeated MessageIdData message_ids = 1;
```

Reply:
The generated java Api will add suffix `List` after the `messageId`, just like `getMessageIdList()`, maybe we don't need to add the suffix `s`.